onnxruntime/test/contrib_ops/matmul_integer_to_float_test.cc

```diff
@@ -82,28 +82,29 @@ void TestMatMulIntegerToFloat(const std::vector<int64_t>& A_dims,
   }
 
   if (has_bias) {
-    test.AddInput<float>("bias", {B_dims.back()}, Bias);
+    test.AddInput<OType>("bias", {B_dims.back()}, Bias);
   } else {
-    test.AddOptionalInputEdge<float>();
+    test.AddOptionalInputEdge<OType>();
   }
 
   test.AddReferenceOutputs(reference_model);
 #if defined(USE_DML)
-  test.SetOutputRelErr("Y", 2e-2f);
+  test.SetOutputRelErr("Y", 1e-2f);
```

Comment:
Currently it looks like ORT tests only use one of the error tolerances, there is no way to use OR condition for when multiple tolerance types are initialized.

Some FP16 test cases fail when we use abs/rel tolerance